screens/Menu/NearbyInterestDetails.js

```diff
@@ -1,33 +1,151 @@
 import React from "react";
-import { View, Text, StyleSheet } from "react-native";
+import { View, Text, StyleSheet, Image, SafeAreaView, AsyncStorage } from "react-native";
 import { Icon, Button } from "native-base";
+import StarRating from "react-native-star-rating";
 
 
+/**
+ * US34 - As a user, I would like to see the nearest outdoor points of interest #14
+ * US35 - As a user, I would like to get the direction to the chosen nearest point of interest #15
+ * US36 - As a user, I would like to see a detailed description of the selected places #29
+ * 
+ * Description: This screen will presents the layout 
+ * for the display of the point of interests. The points 
+ * of interest information will be retrieved from the Google Places API.
+ * The screen is composed on a flatList to create a grid, and have 
+ * an pressable item for each points of interest
+ */
+
 /**Prop passed
  * @param  {} navigation props.navigation is the name of the object from Navigator library
  */
 
 function NearbyInterestDetails(props) {
+/**
+ * navigation that serves to go back to previous screen
+ * @param  {} =>{props.navigation.goBack(
+ */
 
     const goBack = () => {
         props.navigation.goBack();
     }
 
-    const NearbyInterestDetailsScreen = true; 
+    const photoref = props.navigation.getParam("photoref", null);
+    const name = props.navigation.getParam("name", null);
+    const rating = props.navigation.getParam("rating", null);
+    const hours = props.navigation.getParam("hours", null);
+    const address = props.navigation.getParam("address", null);
+    const phone = props.navigation.getParam("phone", null);
+    const web = props.navigation.getParam("web", null);
+    const latitude = props.navigation.getParam("latitude", null);
+    const longitude = props.navigation.getParam("longitude", null);
+    const reviews = props.navigation.getParam("reviews", null);
+
+    // this boolean serves for the back button of the DoubleSearch component 
+    const NearbyInterestDetailsScreen = true;
+    /**
+     * This navigation function allows the navigation and the sending of props to DoubleSearch component
+     * @param  {} =>{props.navigation.navigate("DoubleSearch"
+     * @param  {NearbyInterestDetailsScreen} {NearbyInterestDetailsScreen
+     * @param  {name} name_POI
+     * @param  {latitude} latitude_POI
+     * @param  {longitude} longitude_POI
+     * @param  {} }
+     */
     const goToDoubleSearch = () => {
-        props.navigation.navigate("DoubleSearch", {NearbyInterestDetailsScreen: NearbyInterestDetailsScreen});
+
+        props.navigation.navigate("DoubleSearch", {
+            NearbyInterestDetailsScreen: NearbyInterestDetailsScreen,
+            name_POI: name,
+            latitude_POI: latitude,
+            longitude_POI: longitude,
+        });
     }
 
+    const opening = (hours === true ? "Open Now" : "Close Now");
+
     return (
```

Comment:
## UI inconsistency
I have an iPhone 6S+ and this is the screen I have...
The obvious issue with that is the fact that you don't use React Boostrap enough, meaning, you have a lot of `absolute` positioning.
## Screenshot
![image](https://user-images.githubusercontent.com/33582344/79295642-2a261b00-7ea7-11ea-8b86-7fa3cc5a2e4b.png)

Reply:
The total number review label was added last, it may be missing constraints.

However there's no absolute positioning for the rest of the labels. The reason why it is like that its because it the labels and the background Image are not on the same view.

I can fix it!

Reply:
ahhhh nice!

Reply:
Done!

Reply:
**Unfortunately, I have the same problem...** 

![image](https://user-images.githubusercontent.com/33582344/79403742-b6931500-7f5d-11ea-92a1-ca777befcea6.png)